Update/wata_001.txt

```diff
@@ -5096,39 +5097,39 @@ void main()
 //　でもどうせ魅音の勝ちだ！＠　無敗の帝王＠　俺は魅音に賭けるぞ！＠
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　でもどうせ魅音の勝ちだ！！",
-		   NULL, "Mion's going to win anyways!!", Line_WaitForInput);
+		   NULL, "\"Mion's going to win anyways!!", Line_WaitForInput);
 	OutputLine(NULL, "　無敗の帝王！",
 		   NULL, " She's the Undefeated Empress!", Line_WaitForInput);
 	OutputLine(NULL, "　俺は魅音に賭けるぞ！！",
-		   NULL, " I'm betting on Mion!!", GetGlobalFlag(GLinemodeSp));
+		   NULL, " I'm betting on Mion!!\"", GetGlobalFlag(GLinemodeSp));
 	if (GetGlobalFlag(GADVMode)) { ClearMessage(); } else { OutputLineAll(NULL, "\n", Line_ContinueAfterTyping); }
 
 
 //　いいや神速のレナだ！＠
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　いいや神速のレナだ！！",
-		   NULL, "No, it's going to be Rena with that ungodly speed!!", GetGlobalFlag(GLinemodeSp));
+		   NULL, "\"No, it's going to be Rena with that ungodly speed!!\"", GetGlobalFlag(GLinemodeSp));
```

Comment:
Those are the cheers of the crowd, so I thought it would make it more clear with quotation marks, because it is easily confounded with Keiichi's thoughts...

Reply:
I know but it's again not something uncommon in Higurashi. It's a bit confusing either way - either you don't have quotes for something said by someone or you don't have voices for something in quotes. I think we should keep it as it was.